runhouse/resources/hardware/cluster.py

```diff
@@ -793,6 +794,7 @@ def put(self, key: str, obj: Any, process: str = None):
         """
         if self.on_this_cluster():
             return obj_store.put(key, obj, process=process)
+
```

Comment:
can we revert all these other formatting changes in the file to be consistent with main?